R/dose_grid_search.R

```diff
@@ -82,13 +83,6 @@ dose_grid_search <- function(
     }
   }
 
-  target_time <- get_sampling_times_from_scheme(
```

Comment:
target_time needs to be calculated separately for each interval, and not as a single value passed to `mclapply()`, so moved into the function used in the apply.